source/README.adoc

```diff
@@ -88,7 +88,7 @@ Sources are Kubernetes objects. In addition to the standard Kubernetes
 | `queue_config.delete_when_unused` {optional} | Boolean
 | `queue_config.exclusive` {optional} | Boolean
 | `queue_config.nowait` {optional} | Boolean
-| `channel_config.prefetch_count` {optional} | Int that limits the https://www.rabbitmq.com/consumer-prefetch.html[Consumer Prefetch Value]. Default value is 1. Value msut be between 1 and 1000.
+| `channel_config.prefetch_count` {optional} | Int that limits the https://www.rabbitmq.com/consumer-prefetch.html[Consumer Prefetch Value]. Default value is 1. Value must be between 1 and 1000. With a value of 1 the RabbitMQ Source behaves a FIFO queue, values above 1 are more performance oriented.
```

Comment:
Is there a `Trigger` / `Dispatcher` equivalent that would benefit from the same README update?

Reply:
The format of the samples and the readme's are all over the place, ill create an issue to have a standard

Reply:
OK!

Reply:
Can we add some more color around what "performance oriented" means, i.e. no ordering guarantees?

Reply:
Sure thing!